core/src/main/java/software/amazon/awssdk/core/client/builder/SdkDefaultClientBuilder.java

```diff
@@ -64,27 +63,29 @@
  */
 @SdkProtectedApi
 public abstract class SdkDefaultClientBuilder<B extends SdkClientBuilder<B, C>, C> implements SdkClientBuilder<B, C> {
-    private static final SdkHttpClientFactory DEFAULT_HTTP_CLIENT_FACTORY = new DefaultSdkHttpClientFactory();
-    private static final SdkAsyncHttpClientFactory DEFAULT_ASYNC_HTTP_CLIENT_FACTORY = new DefaultSdkAsyncHttpClientFactory();
+    private static final SdkHttpClient.Builder DEFAULT_HTTP_CLIENT_FACTORY = new DefaultSdkHttpClientFactory();
```

Comment:
can we rename the variables to match with the types?

Reply:
👍